### PR TITLE
Delay datanode startup until namenode FS is initialized

### PR DESCRIPTION
**Pull request by tgjonestx**

### CODE REVIEW

**Comment by tgjonestx** [comment]
retest this

**Comment by tgjonestx** [comment]
retest fix

**Comment by tgjonestx** [comment]
retest this

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -62,6 +55,8 @@ case "$MODE" in
     $HADOOP_PREFIX/bin/hdfs namenode
     ;;
   datanode)
+    wait_for_init_hdfs.sh
+    sleep 30
```
Comment (jafcrocker) [inline scripts/bootstrap.sh]: Doh!

**Pull request merged**